src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -447,13 +470,23 @@ public final SAMSequenceDictionary getReferenceDictionary() {
     }
 
     /**
+     * Returns the master sequence dictionary if it has been set, otherwise null.
+     * In practice, this should only be called after engine initialization in {@code onStartup()}
```

Comment:
Use a javadoc link to the `onStartup()` method: `{@link #onStartup()}` instead of `{@code onStartup()}`